internal/condition/build_condition.go

```diff
@@ -22,7 +23,7 @@ func NewBuildCondition(idleAfter time.Duration) Condition {
 func (c *BuildCondition) Eval(object interface{}) (bool, error) {
 	u, ok := object.(model.User)
 	if !ok {
-		return false, errors.New(fmt.Sprintf("%T is not of type User", object))
+		return false, fmt.Errorf("%T is not of type User", object)
```

Comment:
Do you know the motivation for Errorf over errors.New?

Reply:
It seems like `fmt.Errorf(.....)` is same as `errors.New(fmt.Sprintf(......))`
From golang documentation 
fmt.Errorf() :  It formats a string according to Printf's rules and returns it as an error created by errors.New.
https://blog.golang.org/error-handling-and-go

Reply:
Sure the net effect is exactly the same, otherwise, I would be concerned. The question is why "Effective Go" (which is the foundation for this linting) prefers one over the other.